slack-bridge/config/slack-registration.yaml

```diff
@@ -0,0 +1,9 @@
+id: 547eb430884eae9915e166f386efed45cc931c132b124fa8d72b58ef6af71735
```

Comment:
I think it would be better to store this file under `synapse/config`, since this is configuration intended for synapse. That's what we do for the dimension application service: `./synapse/config/appservice-dimension.yaml`.

Reply:
I felt it was better contained this way. And then it was mounted inside synapse container and utilised. Do you see this approach being better than moving the file under it directly? Also since this file is generated, we might have to share it with dendrite. Keeping it at one place might be better.
But, of course, I would want consistency with how we are already doing it with dimension's config file. Thoughts?

Reply:
I think the structure of this file is specific to synapse, not sure it would work with dendrite. I don't have strong feelings about this, either location works for me :)

Reply:
Yeah, maybe it won't work with dendrite but for consistency sake, should I move this file to match with how things are in dimension? Or make changes on dimension side to be consistent with this approach? Or just let this be as is?

Reply:
In my view, considering this is configuration that defines a synapse module (aka application service), it would make sense to place it under synapse. Then all synapse application services are stored in the same directory.

In the future if we define application services for dendrite, they will likely have a different format, so we can store those under `dendrite/config`, or something similar.

Reply:
Ok, I will make this change